Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -156,6 +156,28 @@ TEST_CASE(test_jpeg_sof2_successive_aproximation)
     EXPECT_EQ(frame.image->size(), Gfx::IntSize(600, 800));
 }
 
+TEST_CASE(test_jpeg_sof1_12bits)
+{
+    auto file = MUST(Core::MappedFile::map(TEST_INPUT("12-bit.jpg"sv)));
```

Comment:
Zombie comment: Do you remember how you created the files added in this commit?

Reply:
Look, there is [this](https://github.com/nico/jpegfiles/blob/4f868c7a6f8f07098548115d7de3dcd46fea1f8b/README.md#yojpg) neat explanation on how it was done. You're very lucky to work with people that meticulous.
Joke aside, I should have put in the commit description.

Reply:
Hahaha 😃 In my memory I had created that file, but then when I looked in git history it was you and I was like "oh, false memory, huh". Thanks for remembering this for me.